src/common/test/db_tests.cc

```diff
@@ -68,13 +68,17 @@ int64_t timeout_handler(event_loop *loop, int64_t id, void *context) {
 TEST object_table_lookup_test(void) {
   event_loop *loop = event_loop_create();
   /* This uses manager_port1. */
-  const char *db_connect_args1[] = {"manager_address", "127.0.0.1:12345"};
+  std::vector<std::string> db_connect_args1;
```

Comment:
an alternative here is to use `std::vector<std::string> db_connect_args1 = {"manager_address", "127.0.0.1:12345"}`